src/devices/lumi.ts

```diff
@@ -1767,10 +1768,13 @@ const definitions: Definition[] = [
         model: 'DJT12LM',
         vendor: 'Aqara',
         description: 'Vibration sensor T1',
-        fromZigbee: [lumi.fromZigbee.lumi_vibration],
```

Comment:
`lumi_vibration` converter is not used anymore (?), if yes, can be removed.

Reply:
I will remove it, once it is confirmed that this extend actually works. The testing is still ongoing